Added documentation for starter workflow (nodejs sample pipeline) (#56)
harness-community/harness-community.github.io: website/docs/CI Starter Workflows/Nodejs-Starter-Workflow.md

# Nodejs Starter Workflow Sample

## Node Example

> This guide covers configuring continuous integration pipelines for a Node starter workflow in Harness CI. If you’re new to Harness CI please read our Onboarding Guide and Nodejs Sample Pipeline Build Configuration Guides first.

Reviewer: ```suggestion
This guide covers configuring continuous integration pipelines for a Node starter workflow in Harness CI. If you’re new to Harness CI, start with Onboarding Guide and Nodejs Sample Pipeline Build Configuration Guides.
```

Reviewer: @codewdhruv hyperlink `Onboarding Guide` and `Nodejs Sample Pipeline Build Configuration` Guides.
